xml-compare/src/test/java/tests/TaskTwoRunTest.java

```diff
@@ -0,0 +1,62 @@
+package tests;
+
+import listeners.TestNGLogListener;
+import lombok.NonNull;
+import lombok.extern.log4j.Log4j2;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+import org.xml.sax.SAXException;
+import testdata.TestData;
+import utils.AppProperties;
+import utils.ZipUtils;
+import utils.xml.XmlUtils;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.File;
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Log4j2
+@Listeners(TestNGLogListener.class)
+public class TaskTwoRunTest {
+
+    @BeforeSuite
+    public void setUp() {
+
+        log.info("Setting up test suite: unzip data file archive");
+        String zipDataFile = AppProperties.getProperty("task_two_zip_data_file", null);
+
+        if (zipDataFile == null) {
+            log.info("The property 'zip_data_file' is not set, so data file unzipping was skipped");
+            return;
+        }
+
+        File zipFile = new File(zipDataFile);
+        String goldData = AppProperties.getProperty("task_two_gold_data_folder");
+        String outputDataFolder = AppProperties.getProperty("task_two_output_data_folder");
+        String goldDataRegex = "A\\d?+.xml";
+        String outputDataRegex = "B\\d?+.xml";
```

Comment:
It seems for me that these patterns won't be working for files like A10.xml, A11.xml etc or for A01.xml, A02.xml

Reply:
You are 100% right. Fixed, thank you! :)